pkg/restore/restore.go

```diff
@@ -793,7 +793,7 @@ func (ctx *context) restoreResource(resource, namespace, resourcePath string) (a
 		// and which backup they came from
 		addRestoreLabels(obj, ctx.restore.Name, ctx.restore.Spec.BackupName)
 
-		ctx.log.Infof("Restoring %s: %v", obj.GroupVersionKind().Kind, name)
+		ctx.log.Infof("Verifying %s: %v", obj.GroupVersionKind().Kind, name)
```

Comment:
This isn't a verification. We need to tell the user were are going to attempt to restore the item. The call to `resourceClient.Create()` attempts to create the item in the Kubernetes apiserver. That is why I suggested changing this from "Restoring" to "Attempting to restore".